client/packages/common/src/authentication/api/hooks/usePreference.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from '@openmsupply-client/common';
 import { useAuthApi } from './useAuthApi';
 
-export const usePreference = <T extends PreferenceKey>(pref: T) => {
+export const usePreference = <T extends PreferenceKey>(...prefs: T[]) => {
```

Comment:
maybe should call this usePreferences now? added support for many prefs from the one hook call 🚀